Airports/EGAA/Positions_Mentor.txt

```diff
@@ -1,3 +1,3 @@
 EGAA_APP:Aldergrove Radar:199.998:AARM:A:EGAA:APP:-:-:4260:4277:N054.39.27.000:W006.13.54.000::
```

Comment:
Called Aldergrove radar when providing radar service?

Reply:
this already is set as radar?